### PR TITLE
Update the install instructions to use `install-theos` 

### PR DESCRIPTION
**Pull request by L1ghtmann**
What does this implement/fix? Explain your changes.
---------------------------------------------------
See title

Does this close any currently open issues?
------------------------------------------
No, but it will supersede #20

Any relevant logs, error output, etc?
-------------------------------------
N/A

Any other comments?
-------------------
It's probably a good idea to check the installer on all platforms once more before this is merged. I tested it as I made changes and Uroboro ran it through additional VM tests, but a third once over wouldn't hurt. 

Where has this been tested?
---------------------------
**Operating System:** …

**Platform:** …

**Target Platform:** …

**Toolchain Version:** …

**SDK Version:** …

### CODE REVIEW

File: docs/Installation-Linux.md [hunk 1]
```diff
@@ -9,60 +9,32 @@ This guide will help you install Theos on your Linux machine, Windows machine vi
...
 
-	For this change to take effect, you must restart your shell. Open a new tab and do `echo $THEOS` on your shell to check if this is working.
+	- SUSE-based distros
 
-1. Clone Theos to your device:
+			zypper install bash curl sudo
 
-		git clone --recursive https://github.com/theos/theos.git $THEOS
+	<sup>
+	<sup>1</sup> In order to use <i>sudo</i>, your non-root user will have to be added to the sudoers file (/etc/sudoers). For more information, see <a href=https://wiki.archlinux.org/title/Sudo#Example_entries>here</a>.
```
Comment (uroboro) [inline docs/Installation-Linux.md:35]: This link does not work in the preview. Does markdown work here?

```suggestion
	<sup>1</sup> In order to use <i>sudo</i>, your non-root user will have to be added to the sudoers file (/etc/sudoers). For more information, see [here](https://wiki.archlinux.org/title/Sudo#Example_entries>).
```

File: docs/Installation-iOS.md [hunk 2]
```diff
@@ -9,45 +9,23 @@ This guide will help you install Theos on your jailbroken iOS device.
...
 
-1. Set up the `THEOS` environment variable:
+1. Install the following prerequisites<sup>1</sup> as *root*:
 
-	bash:
+		apt-get install bash curl sudo
 
-		echo "export THEOS=~/theos" >> ~/.profile
+	<sup>
+	<sup>1</sup> In order to use <i>sudo</i>, your non-root user will have to be added to the sudoers file (/etc/sudoers). For more information, see <a href=https://wiki.archlinux.org/title/Sudo#Example_entries>here</a>.
```
Comment (uroboro) [inline docs/Installation-iOS.md:26]: Same issue as in the iOS page

**Review by uroboro — CHANGES_REQUESTED** [review]
Looks good. I checked the preview site and found one issue with links in the subscript tags

**Review by uroboro — APPROVED** [review]

**Review by kirb — APPROVED** [review]
LGTM, I want to get some mileage out of the installer ASAP so I think it's worth merging this now.

**Pull request merged**